src/relay/qnn/util.h

```diff
@@ -67,6 +69,24 @@ static inline const int32_t GetQmax(const DataType& dtype) {
   }
 }
 
+Expr RequantizeLower(const Expr& input_tensor, const RequantizeAttrs* param,
+                     const Array<IndexExpr>& input_shape, const DataType& out_dtype);
+
+static inline Expr Requantize(const Expr& data, const Array<IndexExpr>& input_shape,
+                              const double& input_scale, const int32_t& input_zero_point,
```

Comment:
no need to have `const xx&` for built-in types, just `double input_scale`, `int32_t input_zero_point` etc.